NewEvaluation.html

```diff
@@ -0,0 +1,674 @@
+ <!DOCTYPE html>
```

Comment:
Similar to my question regarding `Evaluation.css` vs `NewEvaluation.css`. Going to skip review until this is properly cleaned up.